### PR TITLE
Update to monaco-vscode-api 4.5.2

### PR DESCRIPTION
**Pull request by kaisalmen**

### CODE REVIEW

File: packages/wrapper/src/editorAppClassic.ts [hunk 1]
```diff
@@ -41,7 +41,10 @@ export class EditorAppClassic extends EditorAppBase {
     }
 
     override async specifyServices(): Promise<monaco.editor.IEditorOverrideServices> {
-        return Promise.resolve({});
+        const getMonacrhServiceOverride = (await import('@codingame/monaco-vscode-monarch-service-override')).default;
+        return {
+            ...getMonacrhServiceOverride()
```
Comment (CGNonofr) [inline packages/wrapper/src/editorAppClassic.ts:46]: Small typo :)

Comment (kaisalmen) [reply packages/wrapper/src/editorAppClassic.ts:46]: Thank you

Comment (kaisalmen) [reply packages/wrapper/src/editorAppClassic.ts]: Done ⬇️

**Comment by kaisalmen** [comment]
@CGNonofr I will wait for the next release.

**Comment by kaisalmen** [comment]
@CGNonofr this is good to go everything works.

**Comment by kaisalmen** [comment]
@CGNonofr update to 5.0.1. Everything looks fine. 👍

**Comment by kaisalmen** [comment]
I will go ahead and merge this. Next release will start only if the other PR is done.

**Pull request merged**